scripts/build/sign-msi.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Helpful links:
+# https://melatonin.dev/blog/how-to-code-sign-windows-installers-with-an-ev-cert-on-github-actions/
+# https://learn.microsoft.com/en-us/windows/msix/desktop/cicd-keyvault
+
+# Install the required tools
+dotnet tool install --global AzureSignTool
+
+# Sign the MSI file
+AzureSignTool sign \
+    -kvu "$AZURE_KEY_VAULT_URI" \
+    -kvi "$AZURE_CLIENT_ID" \
+    -kvt "$AZURE_TENANT_ID" \
+    -kvs "$AZURE_CLIENT_SECRET" \
+    -kvc "$AZURE_CERT_NAME" \
+    -tr http://timestamp.digicert.com \
+    -v "$MSI_PATH"
```

Comment:
So it's that simple?

I really wonder why they use HTTP for the timestamp servers. Especially a cert company... 🤷‍♀️

Reply:
@ReactorScram, I saw your comment and was curious myself.  I found this: https://stackoverflow.com/questions/61552068/is-timestaping-signatures-over-http-vulnerable

Reply:
I guess it makes sense, like you wouldn't need to layer HTTPS around something already signed or encrypted with GPG, but considering how things are post-LE I thought they'd do it just to avoid people wondering